src/dotnet/commands/dotnet-build/IncrementalPreconditions.cs

```diff
@@ -64,7 +64,7 @@ private string PreconditionsMessage()
             log.Append(
                 "Incremental compilation will be automatically enabled if the above mentioned project properties are not used. " +
                 "For more information on the properties and how to address them, please consult:\n" +
-                @"https://github.com/dotnet/cli/blob/master/Documentation/addressing-incremental-compilation-warnings.md");
+                @"https://github.com/dotnet/cli/blob/rel/1.0.0/Documentation/addressing-incremental-compilation-warnings.md");
```

Comment:
@eerhardt eeek, yes, we should. aka.ms links also work. Also, I believe I've added these to the official docs on dotnet.github.io (https://dotnet.github.io/docs/core-concepts/core-sdk/cli/dotnet-build.html), so I will create these aka.ms links and let's replace the above with aka.ms ones. Will paste them as soon as I can get 2-phase auth to work. 😄